integration/query_array_compat_test.go

```diff
@@ -145,6 +139,7 @@ func TestQueryArrayCompatElemMatch(t *testing.T) {
 			},
 		},
 		"GtZeroWithTypeString": {
+			// A document like {"v":[42, "foo"]} matches this filter (there is an elem >0 and an elem of type string)
```

Comment:
I need a bit more context on this. If I understand correctly `Composites` dataset is not used for tigris handler, not quite sure where `{"v":[42, "foo"]}` comes from :thinking:

Reply:
The filter is set as
```
					{"$elemMatch", bson.D{
						{"$gt", int32(0)},
					}},
					{"$type", "string"},
```

it needs one element to match >0 and one element to be a string. It's not possible to define such array in Tigris. For PG, a case with smth like `{"v":[42, "foo",nill]}` would match.

Reply:
Thanks, got it!